Core/clim-basic/repaint.lisp

```diff
@@ -93,8 +93,8 @@
     (everywhere-region
```

Comment:
"referencign" is a typo in the commit message.

"because sheet may not have any associated at the time being." -> "because the sheet may not have an associated medium at that time."

Reply:
fixed